### ISSUE TITLE
deactivating and activating of modules with screenreaders not possible

### ISSUE THREAD

**Issue by snape84** [contao/core#8391]
The checkbox to activate/disable modules in themes -> layout -> page layouts -> included modules (eingebunden Module) is not visible for screenreaders, because the corresponding checkbox is hidden with display:none in the class mw_enable. The display:none-attribute is interpreted by screenreaders. If elements are hidden with display:none, they are hidden for screnreaders too.

**Comment by NinaG** [comment]
display: none should not be used (like @snape84 mentioned). 

Better: We've got the .invisible-class - it puts content outside of the visible area, but makes it still accessible to screenreaders.

**Comment by leofeyer** [comment]
Fixed in e21e8cede263700ef05724452d227a0c6eaf9a78.
